Botkit/Microsoft.BotKit/BotWorker.cs

```diff
@@ -1,96 +1,107 @@
-// Copyright(c) Microsoft Corporation.All rights reserved.
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
 // Licensed under the MIT License.
 
-using Microsoft.Bot.Builder;
-using Microsoft.Bot.Builder.Dialogs;
-using Microsoft.Bot.Connector;
-using Microsoft.Bot.Schema;
-using Microsoft.BotKit.Core;
-using System;
-using System.Collections.Generic;
-using System.Dynamic;
-using System.Threading.Tasks;
-
 namespace Microsoft.BotKit
 {
+    using System;
```

Comment:
The using statements must be outside the namespace scope. 
I think this rule should be ignored.

Reply:
ACK. I will ignore the rule. Thanks!

Reply:
Done